tests/snmpsim/zynos-mes.snmprec

```diff
@@ -0,0 +1,2 @@
+1.3.6.1.2.1.1.1.0|4|MES Something
```

Comment:
Is this the full output from sysDescr? We require the full output if you can please.